app/brave_settings_strings.grdp

```diff
@@ -1440,6 +1440,17 @@ Leo. This action cannot be undone.
     Data and chat history
   </message>
 
+  <!-- Settings / Playlist -->
+  <message name="IDS_SETTINGS_PLAYLIST_ENABLE_PLAYLIST_LABEL" desc="The text for setting to en/disable Playlist feature">
+    Enable Playlist
+  </message>
+  <message name="IDS_SETTINGS_PLAYLIST_CACHE_BY_DEFAULT_LABEL" desc="The text for setting to en/disable media cache behavior">
+    Automatically keep files for offline
```

Comment:
Suggestion: `Automatically keep files for offline viewing` cc: @rmcfadden3

Reply:
@mkarolin — I believe we need to be careful about words like `keep` that could imply saving or downloading. I'd suggest we stay safe by using `add` instead. So `Automatically add files for offline viewing` … I understand that's slightly odd in this context, but would that work?

Reply:
Works for me! Thanks :)